apps/site/assets/css/_layout-core-shared.scss

```diff
@@ -7,23 +7,23 @@
   }
 }
 
-.body-wrapper {
-  display: flex;
-  flex: 0 0 auto;
-  flex-direction: column;
-  height: 100%;
-  max-width: 100%;
-  overflow: visible;
-  width: 100%;
-
-  @media print {
-    display: block;
-  }
-
-  &.modal-open {
-    overflow: hidden;
-  }
-}
+// .body-wrapper {
+//   display: flex;
+//   flex: 0 0 auto;
+//   flex-direction: column;
+//   height: 100%;
+//   max-width: 100%;
+//   overflow: visible;
```

Comment:
fwiw I was wondering if this issue had to do with a change we did from `overflow: auto` to `overflow: visible` after adding the modal.

iOS mobile Safari really is a troublesome tenant... i.e. this whole spiel: https://www.eventbrite.com/engineering/mobile-safari-why/

Reply:
lol I read that article too